doc/code/qml_logging.rst

```diff
@@ -0,0 +1,9 @@
+qml.logging
+================
+
+.. currentmodule:: pennylane.logging
+
+.. automodapi:: pennylane.logging
+    :no-heading:
+    :include-all-objects:
+    :inheritance-diagram:
```

Comment:
It might be worth giving a once-over of the [supporting classes](https://xanaduai-pennylane--4383.com.readthedocs.build/en/4383/code/qml_logging.html#classes) in the docs to make sure we're happy with how everything is showing up.